docs/source/roadmap.rst

```diff
@@ -1,67 +1,30 @@
-* **Multi target configurations for integration packs:** For a given integration pack, define and manage multiple targets - sets of configurations, so that the user can choose which one to use for a given action.
-* **First class Windows support:** switch to pywinrm for better license. Remote PowerShell via Powershell.REST.API. Windows-native ActionRunners. Windows supported st2workroom.
-* **Projects and Uber-flow:** introduce projects to group and manage rules and workflows. Handle versions and dependencies. "Productize" flow-rule-flow-rule chain pattern, aka "uber-flow". Manage large number of automations across users and teams, on a single StackStorm deployment at enterprise scale.
-* **RBAC refinements and improvements:**
-
-  * tag and property based filters, more refined and convenient access control
-  * permissions on key value objects, arbitrary triggers, support for a default role with is assigned to new users.
-  * WebUI for RBAC configuration
-  * RBAC for ChatOps - allow user to authenticate with StackStorm via bot on chat and when checking permissions directly check permissions of the user who triggered an action / ran a command. Allow introduce a special set of permission types for ChatOps.
-* **StackStorm Forge:** for Community and Enterprise integration and automation packs. Solve the problem of packs spread all over GitHub. Make integration and automation packs discoverable, continuously tested, and community rated
-* **More integration packs:** push more content to the community to help work with most common and widely used tool. Tell us if there is tool you love and think we should integrate with or better yet write a pack yourself.
-* **Action Output Structure Definition**: enable optional definition of action payload, so that it can be introspected and used when passing data between actions in workflows.
-* **Datastore:**  Hierarchical, pluggable and secure datastore to bring together all the gnarly config and key spread in operations environments.
-* **Documentation:** generate REST API docs.
-* **Flow v2:** Visualizing workflow executions. UI improvements, including more "operator" persona features.
-
-
-See :doc:`/changelog` for details on what is done.
-
-.. rubric:: Done in v1.1
-
-* **FLOW:** Visual workflow representation and drag-and-drop workflow designer.
-* **RBAC:** Role based access control for packs, actions, triggers and rules.
-* **Pluggable auth backends** including PAM, Keystone, Enterprise LDAP.
-* **All-in-one installer**: production ready single-box reference deployment with graphical setup wizard.
-* **RHEL 6 and 7 support**
-* **Trace-tags**: ability to track a complete chain of triggers, rules, executions, related to a given triggering event.
-* **Native SSH:** replace Fabric; Fabric based SSH still available and can be enabled via config.
-* **WebUI major face-lift**
-
-
-.. rubric:: Done in v0.11
-
-* **ChatOps:** two-way chat integration beyond imagination.
```

Comment:
> then it's additional work to list major changes
Yeah. And that's what we've not been good at doing. That's why I'd rather remove it from here. Plus roadmap should be where we're *going*, not where we've been. Perhaps we need some sort of "Release History" doc that summarises major changes. Would not include small changes or bugfixes.